app/controllers/admin/budget_investments_controller.rb

```diff
@@ -43,6 +52,42 @@ def toggle_selection
 
   private
 
+    def set_csv
```

Comment:
This method is a bit too large and does not belong in a controller
Could you extract this logic to the investment's model?
Something on these lines http://railscasts.com/episodes/362-exporting-csv-and-excel